lib/checks/visibility/hidden-content.json

```diff
@@ -0,0 +1,12 @@
+{
+  "id": "hidden-content",
+  "evaluate": "hidden-content.js",
+  "metadata": {
+    "impact": "minor",
+    "messages": {
+      "pass": "Page does not have any suspicious hidden content.",
+      "fail": "Manually inspect hidden content on this page.",
```

Comment:
These messages should reflect the intended use:

pass: all the content on the page has been analyzed
fail: there is hidden content on the page that is not being analyzed, you will need to trigger the display of this content to analyze it